applications/sintering/sintering.cc

```diff
@@ -57,7 +57,7 @@ concatenate_strings(const int argc, char **argv)
 {
   std::string result = std::string(argv[0]);
 
-  for (int i = 0; i < argc; ++i)
+  for (int i = 1; i < argc; ++i)
```

Comment:
there is also a boost function for this

Reply:
Do you mean `boost::algorithm::join`? Yea, but that needs a vector of string to be created first, implying again a loop over argv, so leading to more or less the same code after call.

Reply:
Ah. Ok!